src/node.cc

```diff
@@ -3078,6 +3119,14 @@ void SetupProcessObject(Environment* env,
   process->Set(FIXED_ONE_BYTE_STRING(env->isolate(), "execArgv"),
                exec_arguments);
 
+  // process.envFlags
+  Local<Array> env_flags = Array::New(env->isolate());
```

Comment:
Preallocate the array by passing the size as the second argument.

Reply:
will do.  I found another place where it wasn't preallocated (involving another `vector`, `preload_modules`, so this is why I removed it.